ts/serve-static-files/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "encore-ts-static-files",
+  "private": true,
+  "version": "0.0.1",
+  "description": "Serving static files with Encore.ts example",
+  "license": "MPL-2.0",
+  "type": "module",
+  "scripts": {
+    "test": "vitest"
+  },
+  "devDependencies": {
+    "@types/node": "^20.5.7",
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "encore.dev": "^1.40.5"
```

Should make this v1.41.1 to include the SDK for static files
```suggestion
    "encore.dev": "^1.41.1"
```